topo_processor/data/data_transformers/data_transformer_imagery_historic.py

```diff
@@ -38,7 +38,12 @@ def transform_data(self, item: Item) -> None:
                 return
             output_path = os.path.join(item.collection.get_temp_dir(), f"{ulid.ULID()}.tiff")
 
-            create_cog(asset.source_path, output_path).run()
+            try:
+                test = create_cog(asset.source_path, output_path).run()
```

Comment:
If there is no use for the variable `test` I would suggest to not declare it

Reply:
Oops left over from testing, thank you.